src/components/Publish/_constants.tsx

```diff
@@ -53,6 +53,7 @@ export const initialValues: FormPublishData = {
   },
   metadata: {
     nft: { name: '', symbol: '', description: '', image_data: '' },
+    transferable: true,
```

Comment:
would seem logical to put it under `metadata.nft` but this would have had some more refactoring implications as what we mean with `metadata.nft` is capturing the metadata of the NFT alone which we then put in place for OpenSea and the like, and adding and then removing `transferable` from that object during the process seemed a bit overkill so I just dropped it on here

Reply:
it doesn't matter, this is not a library